python/cudf/cudf/tests/test_setitem.py

```diff
@@ -242,3 +242,51 @@ def test_categorical_setitem_invalid():
             "the categories first",
         ):
             gs[0] = 5
+
+
+def test_series_slice_setitem_list():
+    actual = cudf.Series([[1, 2], [2, 3], [3, 4], [4, 5], [6, 7]])
+    actual[slice(0, 3, 1)] = [10, 11]
```

Comment:
Are the nested cases relevant here as well? e.g. `[[10, 11], [12, 23]]`, `{"a": {"b": 100}}`?

Reply:
It's good to have them, Updated the tests to have nested data.